css/conversationView.scss

```diff
@@ -666,6 +666,18 @@ svg {
     color: #3b5998;
   }
 
+.emailButton {
+  cursor: pointer;
+  border: 0px;
+  color: white;
+  background: darkred;
+  font-family: "Helvetica Neue", Helvetica, sans-serif;
```

Comment:
@urakagi Suggest using `$sans-font` sass variable here.

Reply:
This is written by our designer, I am not clear about this but if you think that's better I think you can just go ahead and I'll merge back.